katalog/logging-operated/README.md

```diff
@@ -0,0 +1,39 @@
+# Logging Operated
+
+<!-- <KFD-DOCS> -->
+
+The Logging operated package deploys the fluentd and fluentbit stack via Logging operator CRDs.
+
```

Comment:
there's a minio instance being deployed that is not explained what it does or if it is mandatory
(I remember from the demo that you did that this was to save the error logs, but we should say it somewhere)